test/remote_config/test_remote_config.cpp

```diff
@@ -268,6 +295,25 @@ REMOTE_CONFIG_TEST("response processing") {
           CHECK(!config_state.contains("apply_error"));
         }
       }
+
+      // Verify `cached_target_files` is reported
+      auto cached_target_files = payload.find("cached_target_files");
+      REQUIRE(cached_target_files != payload.end());
+      REQUIRE(cached_target_files->is_array());
+      REQUIRE(cached_target_files->size() == 3);
+
+      const auto ctf = cached_target_files->at(0);
+      CHECK(ctf.at("path").get<std::string_view>() ==
```

Comment:
tests are failing. You need a sort here

Reply:
fixed in 4d15b7dd0a086b25c8bf6956edf42ee558895a3f